static/src/javascripts/projects/common/views/acquisitions-epic-testimonials-prominent.html

```diff
@@ -0,0 +1,41 @@
+<div class="contributions__epic" data-component="<%=componentName%>">
+    <div>
+        <h2 class="contributions__title contributions__title--epic">
+            Why our readers are supporting us:
+        </h2>
+        <div class="epic__testimonial__container">
+            <p class="epic__testimonial__text">
+                I read it fairly often and enjoy the variety of news and entertainment content.
+                It never occurred to me that you wanted contributions, but I'd like you to stick around.
+                Sometimes you actually have to pay for stuff you like. Go figure.
+            </p>
+            <p class="epic__testimonial__name">
+                Mike L
+            </p>
+        </div>
+        <p class="contributions__paragraph contributions__paragraph--epic">
+            More people are reading the Guardian than ever, but far fewer are paying for it. Advertising revenues across the media are falling fast. And <span class="contributions__highlight">unlike many news organisations, we haven’t put up a paywall – we want to keep our journalism as open as we can</span>. So you can see why we need to ask for your help. The Guardian’s independent, investigative journalism takes a lot of time, money and hard work to produce. But we do it because we believe our perspective matters – because it might well be your perspective, too.
+        </p>
+        <p class="contributions__paragraph contributions__paragraph--epic">
+            If everyone who reads our reporting, who likes it, helps to support it, our future would be much more secure.
+        </p>
+    </div>
+
+    <div class="contributions__amount-field">
+        <div>
+            <a class="contributions__option-button contributions__contribute contributions__contribute--epic contributions__contribute--epic-member contributions__contribute--epic-member-top"
+               href="<%=membershipUrl%>"
+               target="_blank">
```

Comment:
For all `target="_blank"`: https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/